index.html

```diff
@@ -5793,7 +5793,7 @@ <h4>Plain HTML or Polyfill DOM Result of the MathML Quadratic Formula</h4>
 				<div class="note" id="role-none-note-none">
 					<h5>Note regarding the ARIA 1.1 <code>none</code> role.</h5>
 					<p>In ARIA 1.1, the working group introduced <code>none</code> as a synonym to the <rref>presentation</rref> role, due to author confusion surrounding the intended meaning of the word "presentation" or "presentational." Many individuals erroneously consider <code>role="presentation"</code> to be synonymous with <code>aria-hidden="true"</code>, and we believe <code>role="none"</code> conveys the actual meaning more unambiguously.</p>
-					<p>Until implementations include sufficient support for <code>role="none"</code>, web authors are advised to use the <rref>presentation</rref> role alone <code>role="presentation"</code> or redundantly as a fallback to the <code>none</code> role <code>role="none presentation"</code>.</p>
+					<p>Until implementations include sufficient support for <code>role="none"</code>, web authors are advised to use the <rref>presentation</rref> role alone <code>role="presentation"</code>.</p>
```

Comment:
Mentioned by @jnurthen, remove this sentence completely because it's no longer necessary:

```suggestion
```